<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaniWindow</class>
 <widget class="QMainWindow" name="PlaniWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>870</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>870</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Planimetrics</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="CustomLabel" name="imageLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Method:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="methodBox">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Brush Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="brushBox">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>35</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="addremToggle">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Erase mode (E)</string>
              </property>
              <property name="shortcut">
               <string>E</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Draw Polygon (A)</string>
              </property>
              <property name="shortcut">
               <string>A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rectBtn">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Set ROI (R)</string>
              </property>
              <property name="shortcut">
               <string>R</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="segBtn">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Segmentation (S)</string>
              </property>
              <property name="shortcut">
               <string>S</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="floodBtn">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Flood Fill (F)</string>
              </property>
              <property name="shortcut">
               <string>F</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearBtn">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Clear Points (C)</string>
              </property>
              <property name="shortcut">
               <string>C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="helpBtn">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="VAM.qrc">
                <normaloff>:/VAM/Icons/1462037714_information2.png</normaloff>:/VAM/Icons/1462037714_information2.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Threshold: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="threshSlider">
              <property name="minimumSize">
               <size>
                <width>500</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>125</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="msgLabel">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="CustomScrollArea" name="imageScrollArea">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>139</y>
             <width>1159</width>
             <height>364</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Weight:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="weightBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Remaining Images</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="imagesRemList">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>60000</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDone"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious_Image"/>
   <addaction name="actionNext_image"/>
   <addaction name="separator"/>
   <addaction name="actionClear_image"/>
   <addaction name="separator"/>
   <addaction name="actionExport"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="VAM.qrc">
     <normaloff>:/VAM/Icons/1462036176_save_accept.png</normaloff>:/VAM/Icons/1462036176_save_accept.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save planimetrics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPrevious_Image">
   <property name="icon">
    <iconset resource="VAM.qrc">
     <normaloff>:/VAM/Icons/1462036195_arrow-left1.png</normaloff>:/VAM/Icons/1462036195_arrow-left1.png</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="toolTip">
    <string>Display the previous image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNext_image">
   <property name="icon">
    <iconset resource="VAM.qrc">
     <normaloff>:/VAM/Icons/1462036198_arrow-right1.png</normaloff>:/VAM/Icons/1462036198_arrow-right1.png</iconset>
   </property>
   <property name="text">
    <string>Next image</string>
   </property>
   <property name="toolTip">
    <string>Display the mext image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClear_image">
   <property name="icon">
    <iconset resource="VAM.qrc">
     <normaloff>:/VAM/Icons/1462036280_trashcan-delete2.png</normaloff>:/VAM/Icons/1462036280_trashcan-delete2.png</iconset>
   </property>
   <property name="text">
    <string>Clear image</string>
   </property>
   <property name="toolTip">
    <string>Clear the current segmentation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionDone">
   <property name="icon">
    <iconset resource="VAM.qrc">
     <normaloff>:/VAM/Icons/1466084211_Accept.png</normaloff>:/VAM/Icons/1466084211_Accept.png</iconset>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="toolTip">
    <string>Return to the main window</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="VAM.qrc">
     <normaloff>:/VAM/Icons/1462455465_centimeter.png</normaloff>:/VAM/Icons/1462455465_centimeter.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export planimetrics data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomScrollArea</class>
   <extends>QScrollArea</extends>
   <header>markerwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VAM.qrc"/>
 </resources>
 <connections/>
</ui>
